conbench/templates/index.html

```diff
@@ -121,6 +121,6 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
 });
 
 enable_search_query_string('#runs');
-
+free_text_column_search('#runs', [1,3,4]);
```

Comment:
I wouldn't mind being able to search the commit message too. I noticed that column is skinnier now, so maybe the search box would widen it a little too?

Reply:
Do we have instances when we had to query the "commit message" col?
Trying to stretch my understanding here.

Reply:
CSS is updated, commit col maintains it's width.